docs/event/metadata.md

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_position: 2
+---
+
+# Metadata
+
+(🟦) Metadata fields marked with this symbol are *Opencast-managed*: they are read-only for users/external applications. All other fields can be freely changed, as long as validity checks pass.
+
+#### General
+- `id: ID` 🟦: unique among all events.
+- `title: NonBlankString`: a short title that is the main label associated with this event for users. Plain text.
+- `description: string?`: user-specified, human-readable description, potentially quite long.
+  - TODO: Decide whether this is plain text, markdown or anything else. External apps displaying this need to know that. Some basic formatting options might be nice?
+- `creators: NonBlankString[]`: The people mainly responsible for creating this video and/or presenting the talk which this video is a recording of. Should contain human-readable names and not usernames. Plain text. This is the main "who?"-information shown in the UIs; other fields in `extraMetadata` (e.g. `dct:contributor`) might be shown too, but less prominently.
+- `language: LangCode?`: describes the (main) language of this event and its metadata. For example, the audio language and (if applicable) language of video content is more important than the language of available subtitles. Generally, assets can have their own language specified.
+- `series: SeriesID?`: optional ID of the series this event belongs to. Must be a valid series ID of an existing series at all time.
+- `owner: Username`: TODO figure out details
+
+#### Time-data
+- `startTime: DateTime?`: Actual real life datetime when the video recording started or will start, with timezone. If this is not applicable, for example because it's a short movie, this should be undefined. UIs should use this as primary date to show for a video and if unset, fallback to `created`.
+- `endTime: DateTime?`: Like `startTime`, but when the video recording stopped. Due to cutting, recording pauses and etc, the `duration` is not necessarily `end - start`.
+- `duration: Milliseconds` 🟦: duration of the event. As specified in ["assets"](./assets), this needs to always match the duration of all non-internal tracks.
+- `updated: Timestamp` 🟦: Timestamp of when anything about this event was last changed.
+- `created: Timestamp` 🟦: Timestamp of when the event was created in Opencast. It is set once when the event is first stored in Opencast's DB, and never changed again. This also implies that scheduled event's `created` date is when the scheduling took place, _not_ the time it is scheduled for (that would be `startDate`)
```

Comment:
@KatrinIhler wrote:

> - suggestion: add ingested additionally to created?
>    - more (explicitely named) date fields = less confusion & discussion about what they mean?

Reply:
Are you suggesting two fields that do exactly the same? Best to start a separate thread as a line comment on the metadata PR.